TodoListDaemonWithCert/Program.cs

```diff
@@ -144,27 +165,34 @@ static int Main(string[] args)
                 }
 
             } while ((retry == true) && (retryCount < 3));
+            return result;
+        }
+
 
+        /// <summary>
+        /// Post a Todo item to the TodoList service
+        /// </summary>
+        static async Task PostNewTodoItem()
```

Comment:
> static async Task PostNewTodoItem() [](start = 7, length = 36)

Should below functions be moved to a separate class file handling todolist stuff?